### ISSUE TITLE
Support Julian and other non-Gregorian calendars in date inputs

### ISSUE THREAD

**Issue by arkhipov** [gramps-project/gramps-web#273]
Hello! First, thanks a lot for a great application. I had been looking forward to something like that for a long time since the Gramps desktop application had appeared.

In the Russian Empire (and probably in other countries with the same religion), the Orthodox Church used the Julian calendar until about the year of 1920. The desktop application allows to choose a calendar in every date input. At the moment, Gramps.JS shows the correct calendar name when you export data from the desktop application, but it does not allow me to enter new dates and edit already entered dates using that calendar.

It would be great to support that in the web application as well. If you give me some hints and suggestions, I can implement this feature on my own and send you a patch.

**Comment by DavidMStraub** [comment]
Yes, that makes sense! You only have to edit the file GrampsjsFormSelectDate.js. You can look at how the other dropdowns are handled to get an idea how it works. Let me know if you have questions.

We should then disable the date pickers if a calendar other the Gregorian is selected, since the browser built in date pickers don't support other calendars.

**Comment by DavidMStraub** [comment]
Yes, that makes sense! You only have to edit the file GrampsjsFormSelectDate.js. You can look at how the other dropdowns are handled to get an idea how it works. Let me know if you have questions.

We should then disable the date pickers if a calendar other the Gregorian is selected, since the browser built in date pickers don't support other calendars.

**Comment by Godzie** [comment]
So sad it is not implemented yet. Most wanted feature. 95% of event is not Gregorian dates.
